.github/dependabot.yml

```diff
@@ -112,6 +111,58 @@ updates:
       time: "00:17"
       timezone: "US/Eastern"
     rebase-strategy: "disabled"
+    groups:
+      babel:
+        patterns:
+          - "*babel*"
+      mdx:
+        patterns:
+          - "*mdx*"
+      storybook:
+        patterns:
+          - "*storybook*"
+      testing-library:
+        patterns:
+          - "@testing-library/*"
+      eslint:
+        patterns:
+          - "*eslint*"
+      jest:
+        patterns:
+          - "*jest*"
+      msw:
+        patterns:
+          - "*msw*"
+      prettier:
+        patterns:
+          - "*prettier*"
+      typescript:
+        patterns:
+          - "typescript"
+          - "tslib"
+          - "ts-node"
+      react:
+        patterns:
+          - "react"
+          - "react-dom"
+      react-router:
+        patterns:
+          - "*react-router*"
+      cypress:
+        patterns:
+          - "*cypress*"
```

Comment:
"`cypress*`"?
There is also an `eslint` match.

Reply:
Eslint will ignore eslint-cypress now